sbi/inference/snpe/snpe_base.py

```diff
@@ -436,6 +445,15 @@ def _loss(self, round_: int, theta: Tensor, x: Tensor, masks: Tensor,) -> Tensor
 
         return -(self.calibration_kernel(x) * log_prob)
 
+    def _warn_if_retrain_from_scratch_snpe(self):
+        if self._retrain_from_scratch_each_round:
+            warnings.warn(
+                "You specified `retrain_from_scratch_each_round=True`. For "
+                "SNPE, we have experienced very poor performance in this "
+                "scenario and we therefore strongly recommend "
+                "`retrain_from_scratch_each_round=False`."
```

Comment:
Maybe link to an issue? We can document there what we learn but perhaps also get valuable feedback. I think it would be enough to put it in the format GH #XXY, advanced users know what to do with that.